src/content/guides/caching.md

```diff
@@ -116,13 +117,14 @@ W> Output may differ depending on your current webpack version. Newer versions m
 
 ## Extracting Boilerplate
 
-As we learned in [code splitting](/guides/code-splitting), the [`CommonsChunkPlugin`](/plugins/commons-chunk-plugin) can be used to split modules out into separate bundles. A lesser-known feature of the `CommonsChunkPlugin` is extracting webpack's boilerplate and manifest which can change with every build. By specifying a name not mentioned in the `entry` configuration, the plugin will automatically extract what we want into a separate bundle:
+As well all know, webpack's boilerplate and manifest change with each build, that manifest and boilerplate generally are what you find in top of your bundle file and having them there can be a little costy.
+
+So with using the new Webpack 4.x we replaced the [`CommonsChunkPlugin`](/plugins/commons-chunk-plugin) for the new [`SplitChunksPlugin`](/plugins/split-chunks-plugin) and way you can use it is via new configuration : `optimization.runtimeChunk` By specifying a name not mentioned in the `entry` configuration, the new optimization will automatically extract what we want into a separate bundle.
```

Comment:
**In webpack** (notice lowercase in `Webpack`'s name) 4+ we replaced X with Y. New way of using it is [`optimization.runtimeChunk`](https://webpack.js.org/configuration/optimization/#optimization-runtimechunk).

No further text is required as the linked section will have all the correct info with example after this is merged:
https://github.com/webpack/webpack.js.org/pull/2112/files#diff-5592ccac75801a9c1ea22366c3ebb7d4R64